Specs/Sequence/English/PhoneNumberModel.json

```diff
@@ -0,0 +1,958 @@
+[
+{
+  "Input": "My phone number is ( 19 ) 38294427.",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "( 19 ) 38294427",
+      "TypeName": "phonenumber",
+      "Resolution": {
+        "value": "( 19 ) 38294427"
```

Comment:
I think the format of Resolution should always the same. But in this file, some are "( xx ) xxxx", "(xx) xxxxx", "(xx)xxxx-xx"

Reply:
@tellarin

Reply:
@tellarin and I have discussed to leave it as the extracted result for now. And I think that parse it to some normal format need further discussion and a lot of work.

Reply:
For this version, let's not do any formatting of the value. This can come in a future PR.